extensions/ccui/base-classes/UIScale9Sprite.js

```diff
@@ -312,6 +312,10 @@ ccui.Scale9Sprite = cc.Scale9Sprite = cc.Node.extend(/** @lends ccui.Scale9Sprit
     setCapInsets: function (capInsets) {
         var contentSize = this._contentSize;
         var tempWidth = contentSize.width, tempHeight = contentSize.height;
+        this._capInsets.x = capInsets.x;
+        this._capInsets.y = capInsets.y;
+        this._capInsets.width = capInsets.width;
+        this._capInsets.height = capInsets.height;
```

Comment:
UIScale9Sprite is really fragile, please do not change the logic order if not necessary